openy.packages.yml

```diff
@@ -71,6 +71,14 @@ y_layout_builder:
   - ws_lb_tabs
   - lb_testimonials
   - lb_webform
+  - y_camp
+  - y_facility
+  - y_lb_main_menu_cta_block
+  - ws_home_branch
+  - layout_builder_restrictions_by_region
+  - lb_activity_finder
```

Comment:
Also, maybe remove the AF block and only install it when AF is needed? Or is AF installed across the board?

Reply:
Moved to dedicated package https://github.com/YCloudYUSA/yusaopeny/pull/112/commits/d3403c60e80d59af1f2e85a7c1887cc70b438a03